src/main/java/org/fundacionjala/sfdc/pageobjects/SalesForceLogIn.java

```diff
@@ -98,5 +98,13 @@ public void logIn(final String user, final String pass) {
     public boolean getCloudIcon() {
         return CommonActions.getElement(this.cloudIcon).isDisplayed();
     }
+
+    /**
+     * Wait until the page loads.
+     */
+    @Override
+    public void waitUntilPageObjectIsLoaded() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)